lib/experimental/Information/Communities/Post/PostEvent/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { DateAvatar } from "@/experimental/Information/Avatars/DateAvatar"
+import { withSkeleton } from "@/lib/skeleton"
+import { Skeleton } from "@/ui/skeleton"
+
+type PostEventProps = {
+  title: string
+  image?: string
+  place: string
+  date: {
+    hour: string
+    day: number
+    month: string
+  }
+}
+
+export const BasePostEvent = ({
+  title,
+  image,
+  place,
+  date,
+}: PostEventProps) => {
+  return (
+    <div className="flex w-full flex-col gap-1 rounded-xl border border-solid border-f1-border-secondary bg-f1-background-inverse-secondary p-1 shadow">
+      {image && (
+        <img
+          src={image}
+          alt={title}
```

Comment:
Since this is just a presentational image we can omit the "alt". Using `alt=""` will make the image be `role="presentation"`, and it will be ignored by assistive technologies. Since we already have the title in text there is no need to repeat it here